### ISSUE TITLE
No value key in token schema

### ISSUE THREAD

**Issue by cailborg** [tokens-bruecke/figma-plugin#3]
When i run the plugin the schema generates fine - but there is no "value" key for the color tokens
```
{
  "sonic": {
    "color": {
      "background": {
        "button": {
          "primary": {
            "default": {
              "type": "color",
              "description": "",
              "scopes": [
                "SHAPE_FILL",
                "FRAME_FILL"
              ],
              "extensions": {
                "variableId": "VariableID:1:3",
                "aliasPath": "{sonic.color.background.button.primary.default}"
              }
            },
            "hover": {
              "type": "color",
              "description": "",
              "scopes": [
                "SHAPE_FILL",
                "FRAME_FILL"
              ],
              "extensions": {
                "variableId": "VariableID:14:46",
                "aliasPath": "{sonic.color.background.button.primary.hover}"
              }
            }
        }
   }
```

Other tokens (e.g space) work fine
```
"space": {
      "element": {
        "large": {
          "value": "12px",
          "type": "dimension",
          "description": "",
          "scopes": [
            "GAP"
          ],
          "extensions": {
            "variableId": "VariableID:1:4",
            "aliasPath": "{sonic.space.element.large}"
          }
        },
```

**Comment by PavelLaptev** [comment]
@cailborg looks like it's a "nesting" issue. I'll try to reproduce this schema on my side

**Comment by PavelLaptev** [comment]
@cailborg that's odd. Can't reproduce this bug. I recreated the same structure (see the image below).
These missing color values are `alias` values?

<img width="1488" alt="image" src="https://github.com/tokens-bruecke/figma-plugin/assets/18498712/59c66255-0ca0-4292-a588-5a9855ad90fd">

If it's okay for you, we can plan a call and check this issue together. Feel free to write me to pavellaptev.figma@gmail.com

**Comment by cailborg** [comment]
One difference is I don't have a "value" column in my variables table - i'm setting up modes for light and dark
<img width="688" alt="Screenshot 2023-08-17 at 9 51 15 am" src="https://github.com/tokens-bruecke/figma-plugin/assets/5325545/6c1781ae-ac60-4dc3-a4d0-5914028ca4de">

**Comment by cailborg** [comment]
Oh and maybe the fact that they are aliased from another set of variables?

**Comment by PavelLaptev** [comment]
Hm… renaming the `Value` title and add `alias` variables didn't show the bug. 
<img width="1497" alt="image" src="https://github.com/tokens-bruecke/figma-plugin/assets/18498712/a7f0d1d7-b97b-477b-b90b-3ed42d83194b">

**Comment by cailborg** [comment]
Development version didn't change any outputs - are you using `resolveForConsumer` in the Figma API to resolve aliases?
https://www.figma.com/plugin-docs/api/Variable#resolveforconsumer

**Comment by cailborg** [comment]
I have discovered the source of the bug - aliases only work where `base` is  a new collection in the same file.
I was originally aliasing tokens from a variable collection in a different file (as I don't want people applying the base tokens to designs) 
<img width="614" alt="Screenshot 2023-08-17 at 12 35 32 pm" src="https://github.com/tokens-bruecke/figma-plugin/assets/5325545/ec2e0a6d-1f90-4656-9f11-57197937b30c">

**Comment by PavelLaptev** [comment]
@cailborg thanks for investigating this. Will try to fix this today

**Comment by PavelLaptev** [comment]
@cailborg I published a new plugin version 1.2.0
Can you try it out?

The plugin will generate an alias name anyway, but it will be a path to the variable as if it was in the current file.

{
  "button": {
    "background": {
      "type": "color",
      "value": "{colors.light.primary.10}"
    }
  }
}

The plugin wouldn't include the variable into the generated JSON in order to avoid duplicates or conflicts with JSON files you can generate from another Figma files.

So you will need to merge the file with the base variables from one file with another where you use them. Otherwise tools like Style Dictionary wouldn't be able to resolve the aliases.

**Comment by cailborg** [comment]
Working perfectly, thanks!
